pgbot/common.py

```diff
@@ -52,6 +52,11 @@
 TOKEN = os.environ["TEST_TOKEN" if TEST_MODE else "TOKEN"]
 PREFIX = "pd!" if TEST_MODE else "pg!"
 
+DEFAULT_DATETIME_FORMAT = '%a %b %d %Y, %I:%M:%S %p'
+CHECK_MARK = "https://images.emojiterra.com/google/android-10/512px/2611.png"
+X_MARK = "https://datalab.noirlab.edu/images/cross.png"
+INFO_EMOJI = "https://chancelight.com/wp-content/uploads/2020/03/blue-exclamation-point.png"
+
```

Comment:
I dont know if we would use these (not a bad idea) but I dont think these are in the "style" of the bot. `INFO_EMOJI` is not even used as far as I have seen

Reply:
Also, seeing all external links here, wonder whether we could get into legal issues with copyright infringement or something. It's best to not have external links like this.

Reply:
Yes, either we use discord emojis or nothing at all.